packages/amplication-client/src/routes/resourceRoutes.tsx

```diff
@@ -4,17 +4,17 @@ import resourceSettingsRoutes from "./resourceSettingsRoutes";
 
 const resourceRoutes = [
   {
-    path: "/:workspace/:project/:resource/entities",
-    Component: lazy(() => import("../Resource/EntitiesTile")),
+    path: "/:workspace([A-Za-z0-9-]{20,})/:project([A-Za-z0-9-]{20,})/:resource([A-Za-z0-9-]{20,})/entities",
+    Component: lazy(() => import("../Entity/EntityList")),
```

Comment:
This looks strange. According to the path, it seems like you want this page
![image](https://user-images.githubusercontent.com/39680385/182040991-7f7e700f-9e09-45db-b659-268bea643e35.png)

But you load the `EntityList` component

Reply:
@abrl91  this is the path page we open, and the component name is EntityList, looks ok for me, am i missing something else?